javascript/big-practice/src/scripts/controllers/authenticationController.js

```diff
@@ -0,0 +1,17 @@
+export default class AuthenticationController {
+  constructor(model, view) {
+    this.model = model;
+    this.view = view;
+  }
+
+  init() {
+    this.view.bindLogin((email, password) => {
+      this.checkUser(email, password);
+    });
+  }
+
+  async checkUser(email, password) {
+    const isErrors = await this.model.constructor.verifyCredential(email, password);
```

Comment:
It should be isValid, because we will base on this variable to show hide message error